src/index.ts

```diff
@@ -107,14 +106,20 @@ const MATCH_SASS_FILENAME_RE = /\.sass$/,
         const out = JSON.stringify(resolvedCss);
 
         let defaultExport = `""`;
+        let imports = '';
 
         if (rollupOptions.insert) {
+          /**
+           * @see insertStyle.ts for additional information
+           * Let rollup handle import by processing insertStyle as a module
+           */
+          imports = `import ${insertFnName} from '${__dirname}/insertStyle.js';\n`;
```

Comment:
Unfortunately this is not feasible: 

if we use `rollup-plugin-sass/dist/insertStyle` as path `rollup` won't be able to resolve it.
This because `rollup` doesn't resolve imports with node by default, to do so you need `node-resolve` plugin.

<img width="426" alt="image" src="https://github.com/elycruz/rollup-plugin-sass/assets/24919330/2c591d70-42c7-4fa9-88c4-4fe7e6b53cb3">

Since the import is not resolved, `rollup` will not include the insertStyle code in the bundle.

---

On the contrary using `__dirname` will fill the import with an absolute path that `rollup` can resolve correctly. 
`__dirname` points to `dist` folder which also contain `insertStyle`  so there should be no problem using it.
(I already tested this in my app and is working)

Reply:
Right though, to your first point, shouldn't we assume that anyone that wants modules from `node_modules/` to be included in their app bundles, would add additional configuration for said modules to be included? (something like `@rollup/plugin-node-resolve`)

To your second point, you are correct, however, doing this causes the users system path to be revealed in their artifacts when adding `external: [/\/insertStyle\.js$/]` to a rollup config - see [test file (in PR)](https://github.com/elycruz/rollup-plugin-sass/pull/139/files#diff-ec61822d929089017a892b4b047229377c9241f16b43252ab851ae39131cdf40R190).

Reply:
Interestingly the complete file system path is not inlined when using the plugin via `node_modules`.

I did a test on my work app (using `rollup@4.12.1`):
<img width="1188" alt="image" src="https://github.com/elycruz/rollup-plugin-sass/assets/24919330/7fdff431-0cc5-4f40-8e89-14e24a3d783f">

Anyway I'll try to use relative path.  Maybe something could be achieved via [`path.relative`](https://nodejs.org/api/path.html#pathrelativefrom-to)

Reply:
Ok, gotcha, no need though, seems the full path only gets rendered when `external: [/\/insertStyle\.js$/]` is set (in the app's rollup config, which wouldn't make sense if `insertStyle` is required for an app, in this case).  

I would say let's setup an 'examples/using-preserve-modules' app example which we could use to validate the behavior, and allow others to do the same, this way we can ensure that everything will work as expected (opening a ticket and PR for this shortly).

Reply:
Sorry, I must have left the `external` setting during some tryouts I've done earlier 😅.
I'll see if I can remove it when migrating to snapshots.